todocore/src/main/java/com/knighttodo/todocore/service/privatedb/repository/DayRepository.java

```diff
@@ -25,4 +26,6 @@ public interface DayRepository extends JpaRepository<Day, UUID> {
     @Modifying
     @Query("update DayTodo dt set dt.removed = true where dt.day.id=:dayId")
     void softDeleteAllDayTodosByDayId(@Param("dayId") UUID dayId);
+
+    Optional<Day> findDayByDate(@Param("date") LocalDate date);
```

Comment:
I'm not sure you need @Param("date") here, it can be just LocalDate date. You need @Param if you use query but you aren't now

Reply:
Thank you.

Reply:
still not fixed

Reply:
Ok. I misunderstood.